### PR TITLE
Backport pr 708

### PR DESCRIPTION
**Pull request by bitwiseman**
## Backport of PR-708

#708 fixes JENKINS-43802.  This a backport of the that change to the 3.10 line.  It pulls in JENKINS-52964 and requires scm-api v2.6.3. 
@jglick @MarkEWaite

### CODE REVIEW

**Comment by jglick** [comment]
Why is this targeted to `master` if it is a backport? Should it not be targeted to `stable-3.10`?

**Comment by bitwiseman** [comment]
@MarkEWaite @jglick  
Thanks!

File: pom.xml [hunk 1]
```diff
@@ -24,14 +24,14 @@
   <inceptionYear>2007</inceptionYear>
 
   <properties>
-    <revision>3.10.2</revision>
+    <revision>3.11.0</revision>
```
Comment (jglick) [inline pom.xml]: Why this change? This is the branch for 3.10.x.

Comment (MarkEWaite) [reply pom.xml]: A significant bump in the required dependencies seemed like it justified a bump in the minor version.  The prior required scm-api allowed lots of flexibility for the actual version installed (2.2.6 or later).

I thought that was a common pattern in plugins.  If that's not a common pattern, I'm more than happy to discard the change to the plugin minor version.

Comment (jglick) [reply pom.xml]: I do not really care, just pointing out that the branch name `stable-3.10` no longer makes sense if you merge this.

Comment (bitwiseman) [reply pom.xml]: @MarkEWaite  If you create a stable-3.11 branch, I'll retarget this PR.

Comment (jglick) [reply pom.xml]: Or simply rename this branch `stable-3.x`, assuming its purpose is “stuff which we do not want to deliver only into 4.x”.

Comment (MarkEWaite) [reply pom.xml]: stable-3.x branch has been created.  Once this pull request has been targeted at stable-3.x, I'll double check that no other open pull requests are targeting stable-3.10, then delete the stable-3.10 branch.

Comment (MarkEWaite) [reply pom.xml]: Targeted at stable-3.x now

**Comment by bitwiseman** [comment]
@jglick @MarkEWaite Could we merge this and get it released? It would be good to get this available.

**Comment by MarkEWaite** [comment]
I'll try to run my interactive tests on this pull request this weekend to confirm that it is good to go.  I need to create a baseline environment for comparison with git plugin 4.0.0 pre-release anyway, so this is a good excuse to launch a parallel environment with a pre-release of git plugin 3.11.0.

**Comment by MarkEWaite** [comment]
Released in git plugin 3.11.0 on July 27, 2019

**Comment by olivergondza** [comment]
Caused https://issues.jenkins-ci.org/browse/JENKINS-58964.
